src/reducers/project-state.js

```diff
@@ -452,7 +452,7 @@ const onLoadedProject = (loadingState, canSave, success) => {
                 type: DONE_LOADING_VM_WITHOUT_ID
             };
         default:
-            break;
+            return;
```

Comment:
I don't understand this change. It might help to add a comment explaining how you could get into this state, and what you are doing in that case.